product_multi_barcode_constraint_per_company/readme/newsfragmensts/2944.feature.rst

```diff
@@ -0,0 +1 @@
+This is a bridge module between "product_barcode_constraint_per_company" and "product_multi_barcode"
```

Comment:
Should be remove this file. News fragments using for updating module not for creating.